### ISSUE TITLE
🐛 [Bug] Error when compiling NextVit model official implementation

### ISSUE THREAD

**Issue by nicehzj** [pytorch/TensorRT#1627]
##  Bug Description

When I use torch_tensorrt.compile for NextVit model official implementation, it raises an error. But before this, I used Efficientnet_b0 and it works fine.

## To Reproduce

1. use the code below

model = timm.create_model('nextvit_base')
model.proj_head = nn.Sequential(
    nn.Linear(in_features=1024, out_features=1),
)

data = torch.load("/kaggle/input/nextvit-b-1536-weights/nextvit_b_1536_f0.pth")
model.load_state_dict(data['model'])

model.to('cuda:0')

model.eval()
trt_model_fp16 = torch_tensorrt.compile(
    model,
    inputs=[
        torch_tensorrt.Input(min_shape=[1, 3, 1536, 960],
                             opt_shape=[2, 3, 1536, 960],
                             max_shape=[2, 3, 1536, 960],dtype=torch.half)],
    enabled_precisions={torch.half},  # Run with FP16
    workspace_size=1 << 32,
)
torch.jit.save(trt_model_fp16, 'kaggle-nextvit-b-1536-gpu-aug0-01-swa.trt_fp16.ts')
print('trt_fp16 ok') 

## Expected behavior

Generate the model compile file

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0): 1.2.0
 - PyTorch Version (e.g. 1.0): 1.12.1+cu113
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version: Python 3.7.12
 - CUDA version: V11.0.221
 - GPU models and configuration: P100
 - Any other relevant information:

## Additional context

The code is implemented on Kaggle platform.

**Comment by peri044** [comment]
@nicehzj  Can you post the error log ?

**Comment by ynhuhu** [comment]
I also encountered the problem with my nextvit in Kagge. Error info is "Can't redefine method: forward on class: __torch__.torch.nn.modules.dropout.Dropout". I don't know why. And your error log?
